clouddriver-web/clouddriver-web.gradle

```diff
@@ -17,7 +17,7 @@ run {
 configurations.all {
   exclude group: 'javax.servlet', module: 'servlet-api'
   exclude group: "org.slf4j", module: "slf4j-log4j12"
-  resolutionStrategy.force 'com.microsoft.rest:client-runtime:1.0.0-20160309.002843-19'
+  resolutionStrategy.force 'com.microsoft.azure:client-runtime:1.6.5'
```

Comment:
Why do we need to pin this version vs. just inheriting the version that the library provides?

Reply:
You are right, remove the pin here